browser/net/brave_network_delegate_base.cc

```diff
@@ -124,6 +151,10 @@ int BraveNetworkDelegateBase::OnHeadersReceived(
     const net::HttpResponseHeaders* original_response_headers,
     scoped_refptr<net::HttpResponseHeaders>* override_response_headers,
     GURL* allowed_unsafe_redirect_url) {
+  brave::RemoveTrackableSecurityHeadersForThirdParty(request,
```

Comment:
does this need to check `if (!request)` first?

Reply:
done!